presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/CteConsumerMatcher.java

```diff
@@ -42,15 +43,19 @@ public boolean shapeMatches(PlanNode node)
     }
 
     @Override
-    public MatchResult detailMatches(PlanNode node, StatsProvider stats, Session session, Metadata metadata, SymbolAliases symbolAliases)
+    public MatchResult detailMatches(PlanNode node, StatsProvider stats, Session session, Metadata metadata, SymbolAliases symbolAliases, BiMap<String, String> cteNameMapping)
```

Comment:
cteNameMapping contains the 1:1 mapping between cte name in planPattern and cte name in query plan

Reply:
Nit: Can be renammed to cteIdMapping here and elsewhere for clarity